go/cmd/rulesctl/cmd/list.go

```diff
@@ -40,11 +40,13 @@ func List() *cobra.Command {
 				out = rules
 			}
 		} else {
-			out = rules.Find(listOptName)
-			if listOptNamesOnly && out != nil {
+			rule := rules.Find(listOptName)
```

Comment:
See https://github.com/vitessio/vitess/pull/7996/commits/6ab6b47bb907f8815bfee48a2a7aab2250aa534f for why this change